Doppler.ImageAnalyzer.Api/Controllers/ImageAnalyzerController.cs

```diff
@@ -38,7 +62,7 @@ public async Task<ActionResult<Response<AnalysisResultResponse>>> AnalyzeImageLi
 
         var resultId = await SaveResultsAsync(response);
 
-        return HandleResponse(mapImageAnalysisResponseList(response, resultId), "Returned image analysis");
+        return HandleResponse(mapImageAnalysisResponseResponse(response, resultId), "Returned image analysis");
```

Comment:
is the method name right? (response response)

Reply:
Yes, I don't like it, but yes; because it maps a `Response<List<ImageAnalysisResponse>>` :grimacing: ... any suggestion is welcomed.

Reply:
just one response or result response

Reply:
Done: https://github.com/FromDoppler/doppler-image-analyzer/pull/324/commits/59ef3b2c891488b5a4ccb1149146ea0a733ab33a. Do you like it?